README.md

```diff
@@ -48,3 +48,24 @@ dependencies: [
 ```
 
 For detailed usage and API documentation, check [the documentation](https://swiftpackageindex.com/apple/swift-certificates/main/documentation/x509).
+
+## Benchmarks
+
+Benchmarks for `swift-certificates` are in a separate Swift Package in the `benchmarks` subfolder of this repository. 
+They use the [`package-benchmark`](https://github.com/ordo-one/package-benchmark) plugin.
+Benchmarks depends on the [`jemalloc`](https://jemalloc.net) memory allocation library, which is used by `package-benchmark` to capture memory allocation statistics.
+An installation guide can be found in the [Getting Started article](https://swiftpackageindex.com/ordo-one/package-benchmark/1.11.1/documentation/benchmark/gettingstarted#Installing-Prerequisites-and-Platform-Support) of `package-benchmark`. 
```

Comment:
This part is very prone to get outdated. Can we just link to the index page of the documentation here.

Reply:
I have removed the version number and it will now always pointer to the latest version. I really want to keep the direct link to the Getting Started article. I don't think it is very likely they will rename the Getting Started article. The section name  "Installing-Prerequisites-and-Platform-Support" may change but that's fine, the link will still work and point to the top of the getting started page.